### ISSUE TITLE
KeyError: 'accessToken'

### ISSUE THREAD

**Issue by HaodiFan** [acheong08/OpenAIAuth#7]
For step 8 the response is empty. Is there something updated?
![image](https://user-images.githubusercontent.com/39939432/217983740-089a74b2-1ac9-4693-aed1-8eae7af79bfd.png)

**Comment by acheong08** [comment]
https://github.com/acheong08/OpenAIAuth/blob/075080472e460e98e005efc545e385b9953ce9b5/src/OpenAIAuth/OpenAIAuth.py#L319-L338

Print statement got removed

**Comment by PixelChen24** [comment]
In line 348
```python
self.access_token = response.json()["accessToken"]
```
It throws the KeyError: 'accessToken'. When I tried to print the `response.json()`, I just got a empty dict {}. What's wrong?

**Comment by acheong08** [comment]
Did you call `get_access_token()`? It's not part of the "begin"

**Comment by PixelChen24** [comment]
> Did you call `get_access_token()`? It's not part of the "begin"

I'm using the latest [v1.1ProxyAPI](https://github.com/acheong08/ChatGPT/tree/2.0.1). I follow the document to create config.json and just run the command
```python
python3 -m revChatGPT.Proxied
```
Then I met this problem

**Comment by acheong08** [comment]
Might be invalid email/password. Not quite sure the exact reason. Can you send more debug?

**Comment by acheong08** [comment]
Try to get it to print the response

**Comment by PixelChen24** [comment]
> Might be invalid email/password. Not quite sure the exact reason. Can you send more debug?

I can ensure that the email and password are valid. Will it be the reason that my ip address is in Mainland China where chatgpt service is not available?

When I saw the latest release "proxy API", I assumed that this service is provide by a third party proxy server instead of the openai offical server directly. So I assumed that this would work fine for mainland china users. So, is my assumption right?

**Comment by acheong08** [comment]
> I can ensure that the email and password are valid. Will it be the reason that my ip address is in Mainland China where chatgpt service is not available?

Yes. China is blocked by openai

**Comment by acheong08** [comment]
> When I saw the latest release "proxy API", I assumed that this service is provide by a third party proxy server instead of the openai offical server directly. So I assumed that this would work fine for mainland china users. So, is my assumption right?

Login is handled on the client side so I can't see your email/password. That is not proxied

**Comment by PixelChen24** [comment]
> > When I saw the latest release "proxy API", I assumed that this service is provide by a third party proxy server instead of the openai offical server directly. So I assumed that this would work fine for mainland china users. So, is my assumption right?
> 
> Login is handled on the client side so I can't see your email/password. That is not proxied

I see. Thank you very much for your nice work and reply!
